cmd/operator/main.go

```diff
@@ -84,6 +86,8 @@ func main() {
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&healthProbeAddr, "health-probe-bind-address", ":8081", "The address the health probe endpoint binds to.")
 	flag.Var(images, "images", fmt.Sprintf("Full images refs to use for containers managed by the operator. E.g thanos=quay.io/thanos/thanos:v0.33.0. Images used are %v", imagesUsed()))
+	flag.BoolVar(&openShiftEnabled, "openshift.enabled", true, "Enable OpenShift specific features such as Console Plugins.")
```

Comment:
Nur sure we want to enable the feature gate by default. But in order to not delay this further we can flip this and adjust the test setup in a follow up.

Reply:
Let's not forget about it then if we don't merge it with enabled=true by default :)

Reply:
I added a task for that [COO-120](https://issues.redhat.com/browse/COO-120) let me know if the details are correct.